templates/_helpers.tpl

```diff
@@ -563,6 +563,44 @@ securityContext for the statefulset vault container
   {{- end }}
 {{- end -}}
 
+{{/*
+securityContext for the test pod template.
+*/}}
+{{- define "server.test.securityContext.pod" -}}
```

Comment:
Hello @fty4 , why do we need to define new template , as its supposed to use the securityContext of the server statefulSet. why not use "server.statefulSet.securityContext.pod" for server-test pod?

Reply:
Hi @KhizerJaan 
I did that because of the indentation of the template.
But now I just added a new commit to use `nindent` to fix the indentation in the template files.

I hope this is the desired solution.

Reply:
@fty4  Thank you , but can you please check indentation , it does not look right for both server-statefulset and and server-test .

Reply:
When I run a diff between the templated yaml from main and after the fix I get this change:

```diff
381a382
>       
414a416
>           
557a560,565
>   
>   securityContext:
>     runAsNonRoot: true
>     runAsGroup: 1000
>     runAsUser: 100
>     fsGroup: 1000
585a594,596
>       
>       securityContext:
>         allowPrivilegeEscalation: false
```

This means the indentation should be fine.
The only issue is see here is that two new lines are beeing added...
But this is a issue in the whole chart and I am not sure how to fix this.

I guess you mean the indentation in the helpers file - I was required to remove the indentation here to later use different indentation on the statefulset and the test.
This is because both use different indentation length ([6](https://github.com/mercedes-benz/vault-helm/blob/fix/test-securitycontext/templates/server-statefulset.yaml#L56) and [10](https://github.com/mercedes-benz/vault-helm/blob/fix/test-securitycontext/templates/server-statefulset.yaml#L77) for the statefulset and [2](https://github.com/mercedes-benz/vault-helm/blob/fix/test-securitycontext/templates/tests/server-test.yaml#L18) and [6](https://github.com/mercedes-benz/vault-helm/blob/fix/test-securitycontext/templates/tests/server-test.yaml#L47) for the server-test).

Do you agree, @KhizerJaan?

Reply:
[this line](https://github.com/hashicorp/vault-helm/pull/930/files#diff-87d68c754766af8e2e930e653be7e4b75fa0c8bdb187cb1bec293f265d9159ffR535)
&
[this](https://github.com/hashicorp/vault-helm/pull/930/files#diff-0ef176392f0823ab2ed4f8ab706fd9bbd322143d85c35831a4c775136da7718aR56)
would add to indent 14 and same is with others.

Reply:
Actually this does not result in a diff in the templated resources - but for sure it is better to remove them when the indentation is set in another place.

I've added commit 43ca102 for this.